index.html

```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+		
+		<!-- ________ *Vitaliy Fomenko* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Vitaliy Fomenko</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">Don't use Twitter</a>
+          </p>
+          <p class="about">test task</p>
```

Comment:
We don't accept any test bio. Kindly add some info about yourself here.

```suggestion
          <p class="about">About You</p>
```